README.md

```diff
@@ -38,7 +38,9 @@ The source code of the ontology is found in the folder `src/ontology/` <br>
 The main file is `src/ontology/oeo.omn` <br>
 
 All own modules are collected in the folder `src/ontology/edits/` <br>
+To understand the structure of these modules, take a look at the structure diagram (https://user-images.githubusercontent.com/38690039/275459325-1c6eb63d-287a-45b5-a107-839c8c09bfe0.png).<br>
```

Comment:
I thought more of having the image directly in the readme instead of just linking it:

```suggestion
The following diagram illustrates the modular file structure of the OEO. It depicts the import and file hierarchy from external imports (right) to the main file oeo.omn (left).
![Structure of the OEO](https://user-images.githubusercontent.com/38690039/275459325-1c6eb63d-287a-45b5-a107-839c8c09bfe0.png)

```